visualization/autoware_overlay_rviz_plugin/autoware_overlay_rviz_plugin/include/turn_signals_display.hpp

```diff
@@ -52,10 +52,6 @@ class TurnSignalsDisplay
   int current_turn_signal_;    // Internal variable to store turn signal state
   int current_hazard_lights_;  // Internal variable to store hazard lights state
   QImage coloredImage(const QImage & source, const QColor & color);
-
-  std::chrono::steady_clock::time_point last_toggle_time_;
-  bool blink_on_ = false;
-  const std::chrono::milliseconds blink_interval_{500};  // Blink interval in milliseconds
```

Comment:
Also when you make the update, please also make this blink interval `250`.
`500` means it will blink at 1 Hz.
But  it is 2 Hz for most vehicles.
https://www.youtube.com/watch?v=iK2K9e_tcdI